packages/apps/examples/nft-react-hooks-e2e/src/integration/app.spec.ts

```diff
@@ -8,6 +8,43 @@ describe('apps-examples-nft-react-hooks', () => {
     cy.login('my-email@something.com', 'myPassword');
 
     // Function helper example, see `../support/app.po.ts` file
-    getGreeting().contains('Welcome apps-examples-nft-react-hooks');
+    getGreeting().contains('Welcome to GraphHack NFT App!');
+  });
+
+  it('should get rate limited', () => {
+    cy.visit('/collections');
+    cy.intercept('/graphql').as('gqlRequest');
+    cy.get('button').click();
+    cy.wait('@gqlRequest');
+    cy.get('button').click();
+    cy.wait('@gqlRequest');
+    cy.get('button').click();
+    cy.wait('@gqlRequest');
+    cy.get('button').click();
+    cy.wait('@gqlRequest');
+    cy.get('button').click();
+    cy.wait('@gqlRequest');
+    cy.get('button').click();
+    cy.wait('@gqlRequest');
+    cy.get('button').click();
+    cy.wait('@gqlRequest');
+    cy.get('button').click();
+    cy.wait('@gqlRequest');
+    cy.get('button').click();
+    cy.wait('@gqlRequest');
+    cy.get('button').click();
+    cy.wait('@gqlRequest');
+    cy.get('button').click();
+    cy.wait('@gqlRequest');
+    cy.get('button').click();
+    cy.wait('@gqlRequest');
+    cy.get('button').click();
+    cy.wait('@gqlRequest');
+    cy.get('button').click();
+    cy.wait('@gqlRequest');
+    cy.get('button').click();
+    cy.wait('@gqlRequest');
+    cy.get('button').click();
+    cy.wait('@gqlRequest');
```

Comment:
where are we asserting here that the rate limit got triggered?

Reply:
oops that was my bad, I was only able to force the rate limits with some janky extra changes to the example app, will remove this test block altogether, good spot

Reply:
fixed on f8e70c3!